src/microvm/src/vmm/microvm/kvm/emulator.rs

```diff
@@ -36,6 +38,8 @@ pub struct Emulator {
     input: Box<InputFn>,
     /// Output function used for emulating I/O port writes.
     output: Box<OutputFn>,
+    /// Channel to tell the orchestrator all vcpus have paused.
+    _paused_tx: Sender<Message>,
```

Comment:
Please add a comment saying why we need to keep this around. It would help others to accidently removing it from the structure.